cypress/selectors/agenda.selectors.js

```diff
@@ -18,7 +18,7 @@ const selectors = {
   // agendaitem-decision
   agendaitemDecision: {
     uploadFile: '[data-test-agendaitem-decision-upload-file]',
-    edit: '[data-test-agendaitem-decision-edit]',
+    edit: '[data-test-agendaitem-decision] + button',
```

Comment:
@ValenberghsSven Binnen `AuPill` (uit ember-appuniversum) is de action-knop niet meer direct aanspreekbaar binnen de tests. Momenteel heb ik dit op deze manier opgelost, maar het lijkt me wel opportuun om dit eens deftig te bekijken tijdens de komende sprint?

Reply:
We zouden kunnen zoeken op specifiek `au-c-pill-action` (dependency selector), maar op een button kan ook wel. 
Ik kijk wel eens wat het properste is

Reply:
voorlopig ok, zolang het werkt blijven we ermee werken.